reference/latest/src/main/java/com/example/docs/item/FabricDocsReferenceItems.java

```diff
@@ -1,28 +1,35 @@
 package com.example.docs.item;
 
-import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
 import net.minecraft.registry.Registries;
 import net.minecraft.registry.Registry;
+import net.minecraft.text.Text;
 import net.minecraft.util.Identifier;
 
-import net.fabricmc.fabric.api.item.v1.FabricItemSettings;
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.itemgroup.v1.FabricItemGroup;
 
 import com.example.docs.FabricDocsReference;
 
-public class FabricDocsReferenceItems {
-	private FabricDocsReferenceItems() {
-		// private empty constructor to avoid accidental instantiation
-	}
-
-	public static final CustomSoundItem CUSTOM_SOUND_ITEM = register("custom-sound-item",
-			new CustomSoundItem(new FabricItemSettings().maxCount(1)));
-
-	private static <T extends Item> T register(String name, T item) {
-		Identifier identifier = new Identifier(FabricDocsReference.MOD_ID, name);
-		return Registry.register(Registries.ITEM, identifier, item);
-	}
-
-	public static void initialize() {
-		FabricDocsReference.LOGGER.info("initializing items");
+// :::1
+public class FabricDocsReferenceItems implements ModInitializer {
+	// :::1
+	// :::2
+	public static final ItemGroup MY_MOD_ITEMGROUP = FabricItemGroup.builder()
+			.icon(() -> new ItemStack(ModItems.GUIDITE_SWORD))
+			.displayName(Text.translatable("itemGroup.fabric_docs_reference"))
+			.build();
+	// :::2
+	// :::1
+	@Override
+	public void onInitialize() {
+		ModItems.initialize();
```

Comment:
[NIT] Not sure this really needs to be its own class?

Reply:
See point from before.